xAPI.md

```diff
@@ -510,6 +513,69 @@ Clarifications and additional examples including:
 - Addiitional explanitory text and diagrams
 [1.0.2...1.0.3](https://github.com/adlnet/xAPI-Spec/compare/1.0.2...1.0.3)
 
+
+<a name="Appendix1B"/>
+### Appendix B: cmi5 Example 
+
+The following example illustrates a statement following the cmi5 Community of Practice. It demonstrates use of Extensions 
+and "category" Context Activities.
+
+```
+{
+  "id":"2a41c918-b88b-4220-20a5-a4c32391a240",
+  "actor": {
+    "objectType": "Agent",
+    "name": "Gert Frobe",
+    "account": {
+      "homePage": "http://example.adlnet.gov",
+      "name": "1625378"
+    }
+  },
+  "verb": {
+    "id": "http://w3id.org/xapi/verbs/failed",
+    "display": {
+      "en-US": "failed"
+    }
+  },
+  "object": {
+    "id":"https://example.adlnet.gov/AUidentifier",
+    "objectType": "Activity"
+  },
+  "result": {
+    "score": {
+      "scaled": 0.65,
+      "raw": 65,
+      "min": 0,
+      "max": 100
+    },
+    "success": false,
+    "completion": false,
```

Comment:
See https://github.com/AICC/CMI-5_Spec_Current/issues/331 but I'm not sure this makes it into the final specification because the possibilities of pass/fail and complete are just not compatible at the moment.